c/secp256k1_blake160_multisig_all.c

```diff
@@ -115,8 +115,10 @@ int main() {
   }
   // Extract optional since value.
   uint64_t since = 0;
+  bool has_since = false;
   if (args_bytes_seg.size == BLAKE160_SIZE + sizeof(uint64_t)) {
     since = *(uint64_t *)&args_bytes_seg.ptr[BLAKE160_SIZE];
```

Comment:
What if we make `since` a local variable in this `if` branch, and does `check_since` checking here? It seems nowhere else is the `since` value used. In addition, the `has_since` flag can also be skipped.

Reply:
It only means if there is an invalid since value and another potential error, the invalid since value will be picked up first instead of the other potential error.

To me personally, it is an acceptable change. We alter the behavior anyway with any change to the multisig script.